src/lib/stores/events.ts

```diff
@@ -50,6 +50,8 @@ const runId = derived([page], ([$page]) => {
 
 const settings = derived([page], ([$page]) => $page.stuff.settings);
 
+const accessToken = derived([page], ([$page]) => $page.stuff.user?.accessToken);
```

Comment:
Since we're already using stuff everywhere it's cool to let this through but stuff is being deprecated and we probably want to find the better paradigm for it later later. (No work needed just opining)

Reply:
oh i would like to not use stuff and at least store instead, can change that in the PR